### ISSUE TITLE
Pre-defined options for Source and Medium

### ISSUE THREAD

**Issue by chrislarrycarl** [asdfdotdev/utm.codes#47]
Adding the ability for administrators to configure fixed options for source and medium would be helpful in creating links more quickly and consistently.

**Comment by guyeshet** [comment]
I totally agree, having only open options for medium and source is prone to errors

**Comment by RayHollister** [comment]
I agree. One of the reasons I am considering this plugin is because how the users have a terrible time consistently using the medium and source correctly.

**Comment by RayHollister** [comment]
> I agree. One of the reasons I am considering this plugin is because how the users have a terrible time consistently using the medium and source correctly.

An easier solution might be to autocomplete the fields based on previous links that have been saved. Honestly, I don't care if "newspper" is misspelled as long as it is _always_ misspelled! 🤣 At least then it will appear in Google Analytics and in my Google Data Studio reports correctly!
